src/main/resources/templates/home.html

```diff
@@ -48,7 +48,7 @@
                 <option>Pet #3</option>
               </select>
             </div>
-            <img class="img-responsive" src="nolan.PNG" alt="pet #1">
+            <img class="img-responsive" th:src="@{/img/nolan.png}" alt="pet #1">
```

Comment:
I also changed the references from manual filesystem based to Thymeleaf specific pointers. As an aside, this also meant that I could get "nolan.png" to render on the home page when it wasn't visible earlier.